CIScripts/Build/BuildFunctions.ps1

```diff
@@ -196,7 +197,10 @@ function Invoke-ExtensionBuild {
         Copy-Item $utilsMSI $OutputPath -Recurse -Container
         Copy-Item $vRouterMSI $OutputPath -Recurse -Container
         Copy-Item $vRouterCert $OutputPath -Recurse -Container
-        Copy-Item $vTestPath "$OutputPath\utils\vtest" -Recurse -Container
+        New-Item -Type Directory -Path "$OutputPath\utils\vtest" | Out-Null
```

Comment:
In Build.ps1 we have all New-Item stuff. Let's move it there as well.

Reply:
Think of this `New-Item` as an implementation detail of the following `Copy-Item`s (Couldn't do it with a single `Copy-Item -Recurse`, because I needed two different filters, therefore split into separate `New-Item` + two `Copy-Item`s). Do you still think it should be moved to `Build.ps1`?

Reply:
I think the contract about artifact locations should be defined in one place. Why doesn't `-Container` option work here?